### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by BrandyTN2UK**
I've added some of the FAQs from the Discord server to here but need to know if this is the direction you want to take with format and questions.

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
-# Frequenly answered questions
+# Frequently Answered Questions
```
Comment (jgayfer) [inline docs/faq.md:1]: Our titles on all pages are capitalized with only the first word right now. If we want to make this change, I'd like it to be all or nothing.

For this PR, we can probably leave it the same.

**Review by jgayfer — COMMENTED** [review]
Thanks for the PR! 🎉 

All looks reasonable to me, but I'd like to wait for @reiichi001 to weigh in.

File: docs/faq.md (hunk 1 above)
Comment (reiichi001) [reply docs/faq.md:1]: I think page titles would make sense to be in Pascal Case. Especially when we're using acronyms like FAQ. 

It should also help make a logical distinction between pages and subpages (which could be sentences) if we ever needed to break out a dedicated page for a particular entry, but still wanted to keep it listed in its FAQ section.

Comment (jgayfer) [reply docs/faq.md:1]: I'm down to change it 👍 But we should do it for all the pages at once for the sake of consistency.

Comment (BrandyTN2UK) [reply docs/faq.md:1]: Also - Is it "Frequently Asked Questions" or "Frequently Answered Questions"?

**Review by jgayfer — APPROVED** [review]

**Pull request merged**